tests/OSModulesTest.php

```diff
@@ -165,13 +165,6 @@ public function dumpedDataProvider()
 
     private function stubClasses(): void
     {
-        $this->app->bind('log', function ($app) {
-            $mock = \Mockery::mock('\App\Facades\LogManager[event]', [$app]);
-            $mock->shouldReceive('event');
-
-            return $mock;
-        });
-
```

Comment:
@murrant what's left here is this, I suppose we could stub it out in the same way as astext?

Reply:
You can't mock static methods.